### ISSUE TITLE
Conversion error when running the site locally

### ISSUE THREAD

**Issue by devGHBot** [github/choosealicense.com#383]
_Summarized (continued from another topic)_

Let me state my lack of Jekyll know-how and as @mlinksva said this can help for any future contributors.  

Following [Run it on your machine](https://github.com/github/choosealicense.com#run-it-on-your-machine) (Win7) to view some changes for a PR (see #355 ) when executing _script/server_ I get this error:

`Conversion error: Jekyll::Converters::Scss encountered an error while converting 'assets/css/application.scss': Invalid CSS after "...s/normalize.css": expected "{", was "" on line 1`

**Steps taken:**
- Updated all dependencies (Jekyll, ruby, bundler...)
- Went through [Jekyll on Windows by juthilo](http://jekyll-windows.juthilo.com/) (ensuring proper encoding is used in the terminal and the file)
- Tried a vagrant box, while ssh'd the script throws same error [Jekyll on Win via Vagrant](http://www.jamessturtevant.com/posts/running-jekyll-in-windows/)
- Tried to run jekyll serve without any flags and same result
- Removing the yaml front matter block from application.scss, the script works but it doesn't generate a application.css when viewing the site in browser.

Besides throwing windows out the window :+1:
...I'll keep troubleshooting.

**Comment by devGHBot** [comment]
This may not be the proper way to do this because it makes me wonder how you are @mlinksva able to build it without getting an error but I added `@import` to the partials:

In **_sass/normalize.scss**
`../assets/vendor/normalize-css/normalize.css`
to
`@import "../assets/vendor/normalize-css/normalize.css"`

In **_sass/jquery.qtip.scss**
`../assets/vendor/qtip2/jquery.qtip.css`
to
`@import "../assets/vendor/qtip2/jquery.qtip.css"`

**Comment by devGHBot** [comment]
Any feedback on this will be great, I would like to know if I am working with this correctly.

**Comment by mlinksva** [comment]
@LeoFNaN sorry I've not had a chance to test on Windows yet.

But looking closer, I bet the problem is that the files in `_sass` are symbolic links -- "../assets/vendor/normalize-css/normalize.css" and "../assets/vendor/qtip2/jquery.qtip.css" are not supposed to be the contents of the files, but rather the paths the symlinks point to.

Probably something is not supporting symlinks. Are you on a FAT filesystem? See http://stackoverflow.com/questions/11662868/what-happens-when-i-clone-a-repository-with-symlinks-on-windows/11664406#11664406

The symlinks were added in https://github.com/github/choosealicense.com/commit/1058d71b2e9210efb1382f417b4bd15dae6991d7

I attempted a fix in https://github.com/github/choosealicense.com/pull/395

Does that make it work for you?
